cristin-import/src/test/java/no/unit/nva/cristin/CristinMapperTest.java

```diff
@@ -0,0 +1,159 @@
+package no.unit.nva.cristin;
+
+import static nva.commons.core.JsonUtils.objectMapperWithEmpty;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.type.CollectionType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import no.unit.nva.model.AdditionalIdentifier;
+import no.unit.nva.model.EntityDescription;
+import no.unit.nva.model.Publication;
+import no.unit.nva.model.PublicationDate;
+import nva.commons.core.attempt.Try;
+import nva.commons.core.ioutils.IoUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CristinMapperTest {
+
+    public static final String SAMPLE_INPUT_01 = "input01";
+    private static final Integer NUMBER_OF_LINES_IN_RESOURCES_FILE = 100;
+
+    @Test
+    public void mapReturnsResourceWithCristinIdStoredInAdditionalIdentifiers() throws IOException {
```

Comment:
Tests will describe the conversion of one field to another and cover the corner cases.